test/integration/015_cli_invocation_tests/test_cli_invocation.py

```diff
@@ -173,3 +234,41 @@ def _run_simple_dbt_commands(self, project_dir):
         # In case of 'dbt clean' also test that the clean-targets directories were deleted.
         for target in self.config.clean_targets:
             assert not os.path.isdir(target)
+
+
+class TestCLIInvocationWithProfilesAndProjectDir(ModelCopyingIntegrationTest):
+
+    @property
+    def schema(self):
+        return "test_cli_invocation_015"
+
+    @property
+    def models(self):
+        return "models"
+
+    @property
+    def custom_schema(self):
+        return "{}_custom".format(self.unique_schema())
+
+    @use_profile("postgres")
+    def test_sub_command_with_profiles_separate_from_project_dir(
+        self,
+    ):
+        """
+        Test if a sub command runs well when a profiles dir is separate from a
+        project dir.
+        """
+        dbt_sub_command = "deps"
+        profiles_dir = "./tmp-profile"
+        workdir = os.getcwd()
+        with temporary_working_directory() as tmpdir:
+
+            profiles = get_custom_profiles_config(
+                self.database_host, self.custom_schema)
+            create_directory_with_custom_profiles(profiles_dir, profiles)
+
+            project_dir = os.path.relpath(workdir, tmpdir)
+            if os.path.exists(f"{project_dir}/profiles.yml"):
+                os.remove(f"{project_dir}/profiles.yml")
```

Comment:
I do not know where/when the default project directory is created, but it contains a `profiles.yml`. This could be a side effect that effects other tests